README.md

```diff
@@ -44,6 +44,7 @@ LwF | Split MNIST | LwF.test_smnist | Stream Accuracy
 LwF | Permuted MNIST | LwF.test_pmnist | Stream Accuracy
 LwF | Split Tiny ImageNet | LwF.test_stinyimagenet | Stream Accuracy
 GSS | Split MNIST | GSS.test_smnist | Stream Accuracy
+iCaRL | iCIFAR100 | iCARL.test_iCIFAR100_batch10 | Average Incremental Accuracy
```

Comment:
to be more aligned with avalanche, I would call the benchmark `scifar100` (so `test_scifar100`).

Reply:
DONE